api/cisco/nx/v1alpha1/vtepconfig_types.go

```diff
@@ -0,0 +1,81 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and IronCore contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1alpha1 "github.com/ironcore-dev/network-operator/api/core/v1alpha1"
+)
+
+// +kubebuilder:rbac:groups=nx.cisco.networking.metal.ironcore.dev,resources=vtepconfigs,verbs=get;list;watch
+
+// VTEPConfig defines the Cisco-specific configuration of a VTEP/NVE
+type VTEPConfigSpec struct {
+	// AdvertiseVMAC is equivalent to: `advertise virtual-rmac`
+	// +optional
+	AdvertiseVMAC *bool `json:"advertiseVMAC,omitempty"`
```

Comment:
```suggestion
	AdvertiseVirtualMAC *bool `json:"advertiseVirtualMac,omitempty"`
```
According to the kubernetes api conventions, this should be written out.